examples/backup_vm.py

```diff
@@ -221,7 +221,7 @@ def cmd_download(args):
 
         backup = get_backup(connection, backup_service, args.backup_uuid)
         if backup.phase != types.BackupPhase.READY:
-            raise RuntimeError("Backup {} is not ready".format(backup_uuid))
+            raise RuntimeError("Backup {} is not ready".format(args.backup_uuid))
```

Comment:
Maybe worth also adding a current `backup.phase` state.
But that looks good as well.

Reply:
That is a good point! Let's just add it.